src/tcgwars/logic/impl/gen3/PowerKeepers.groovy

```diff
@@ -445,8 +457,9 @@ public enum PowerKeepers implements LogicCardInfo {
         pokeBody "Primal Stare", {
           text "As long as Kabutops is your Active Pokémon, your opponent can't play any Basic Pokémon or Evolution cards from his or her hand to evolve his or her Active Pokémon."
           delayedA {
-            before EVOLVE_STANDARD, self.owner.opposite.pbg.active,{
-              if (bg.currentTurn==self.owner.opposite) {
+            before EVOLVE_STANDARD, {
+              if (ef.pokemonToBeEvolved.owner != self.owner && ef.pokemonToBeEvolved.active && self.active) {
+                wcu "Primal Stare prevents evolving your Active Pokémon"
                 prevent()
```

Comment:
This doesn't seem to handle the case of playing Basic Pokémon.

Reply:
That's... A really weird wording. Is it meant to block Baby Evolution maybe? Will check in a bit.

Reply:
Testing with Dracovish (DAA), that also seems to fail to stop Baby Evolution. Will need to think it a bit more on how to prevent it, seems to be the main case that this is meant to cover re: Basic Pokémon.

Reply:
Probably something best left for using a debugger on to figure out the proper fix honestly.

Reply:
Yup. Will leave a TODO in it.